README.md

```diff
@@ -69,14 +69,26 @@ go build -o dittofs cmd/dittofs/main.go
 ./dittofs start
 ```
 
+### Run with Docker
+
+To run DittoFS is with Docker, ensure first to have the config file `~/.config/dittofs/config.yaml` to let docker compose mount it in the container:
+
+```bash
+# Start the server
+docker-compose up -d
+
+# View logs
+docker-compose logs -f dittofs
+```
+
 ### Mount from Client
 
 ```bash
 # Linux
 sudo mount -t nfs -o nfsvers=3,tcp,port=12049,mountport=12049 localhost:/export /mnt/nfs
 
 # macOS
-sudo mount -t nfs -o nfsvers=3,tcp,port=12049,mountport=12049 localhost:/export /mnt/nfs
+sudo mount -t nfs -o nfsvers=3,tcp,port=12049,mountport=12049 localhost:/export /tmp/nfs
```

Comment:
The point here is not about the `/mnt` folder vs `/tmp` folder. The point is that the folder where you want to mount the volume **must exist**, so it has to be created before. Putting `/tmp` here is ok, but we should also specify that the folder must be created before running the command.